### DISCUSSION TITLE
dapp交互时报错

### DISCUSSION THREAD

**Discussion by xiegudong45** [NonceGeek/Move-Camp#206] [Q&A] [answered]
我在部署合约 & 启动dapp后进行交互，但是到支付gas fee时报错了，这个要怎么解决呢？
![image](https://user-images.githubusercontent.com/30063330/200852896-2c556ac8-d216-417a-945d-ac5257e3e0b8.png)

**Comment by jolestar** [comment, marked as answer]
依赖问题，应该是依赖的 module 没有部署到链上，或者链上的版本不一样。

**Reply by xiegudong45** [reply]
感谢老师，已定位到问题

**Comment by v1xingyue** [comment]
应该是资源查询不到，建议按照如下的几步排查: 

1. 合约没部署成功，检查下是否有合约。
2. 初始化的时候，该地址已经存在resource, 使用命令行确认。
3. 操作资源的时候，该资源不存在，依然使用命令行确认。

**Comment by jolestar** [comment]
资源查不到不应该是 linker error，link error 应该就是 module 没部署，或者部署的有问题
